x-pack/solutions/security/plugins/security_solution/common/siem_migrations/model/common.schema.yaml

```diff
@@ -35,3 +35,22 @@ components:
             dataset:
               type: string
               description: The dataset name to use for evaluations.
+
+    MigrationStatus:
+      type: string
+      description: The status of migration.
+      enum: # should match SiemMigrationsStatus enum at ../constants.ts
+        - pending
+        - processing
+        - completed
+        - failed
+
+    MigrationTaskStatus:
+      type: string
+      description: The status of the migration task.
+      enum: # should match SiemMigrationTaskStatus enum at ../constants.ts
```

Comment:
@semd , do you remember, why do we have a separate `SiemMigrationTaskStatus` and can't we simple reuse `MigrationTaskStatus`.

Reply:
Because the enums created by Zod are not native TS enums, which don't have the same functionality. 
I created [this issue](https://github.com/elastic/security-team/issues/11348) in the security private repo, but I think I'll move it to the public one.